.travis.yml

```diff
@@ -18,13 +18,6 @@ after_failure:
 
 jobs:
   include:
-    - name: "Unit Tests"
-      install:
-        - now install coverage linters pytest cylc rose
-      script:
-        - now test style
-        - now test units
```

Comment:
Why not just copy the `now` commands over to GH actions to save re-implementing everything?

Reply:
Will that work?

Reply:
Yes that will work.

Reply:
I'll give it a go - probably will end up creating a new PR and closing this one. Ah well.

Reply:
What's the motive for this PR?

Reply:
Happy to move over to GH actions, however, it wont help the tests to pass.

Reply:
Tests take a long time to pass or fail. Perhaps your are right - i should get it working on Travis.